index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
You can require a minimum number of characters before and/or after the hyphenation point:

```css
h4 a {
    -webkit-hyphens: auto;
    -ms-hyphens: auto;
    hyphens: auto;

    -webkit-hyphenate-limit-before: 10;
    -webkit-hyphenate-limit-after: 5;
    -ms-hyphenate-limit-chars: auto 10 5;
    hyphenate-limit-chars: auto 10 5;
}
```

* <http://trac.webkit.org/changeset/80438>
* <https://www.w3.org/TR/css-text-4/#hyphenate-char-limits>